README.md

```diff
@@ -55,6 +55,8 @@ Use the following Target IDs (--targetId option) when running commands directly:
 
 | Device name      | Firmware Version                   | Target ID    |
 |------------------|------------------------------------|--------------|
+| `Flex`           | all                                | `0x33300004` |
+| `Stax`           | all                                | `0x33200004` |
 | `Nano S Plus`    | all                                | `0x33100004` |
 | `Nano X`         | < 2.2.1 (**developer units only**) | `0x33000004` |
 | `Nano X`         | \>=  2.2.1                         | `0x33000004` |
```

Comment:
Why keeping 2 different lines for NanoX, whereas the `Target ID` values are the same?

Reply:
true --> fixed